### PR TITLE
Added option for tracers for StorageESP and Search #619

### PR DESCRIPTION
**Pull request by AFelidae**
**Describe the pull**
Option in StorageESP and search modules to have tracers point to blocks.

**Describe how this pull is helpful**
Easier to find blocks you're looking for.

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/module/modules/render/Search.java [hunk 1]
```diff
@@ -270,6 +279,41 @@ private boolean shouldUpdate() {
...
+                .rotateYaw((float) -Math.toRadians(mc.player.rotationYaw));
+
+        float red = ((float) (colour >> 16 & 0xFF)) / 255;
+        float green = ((float) (colour >> 8 & 0xFF)) / 255;
+        float blue = ((float) (colour & 0xFF)) / 255;
+
+        drawLineFromPosToPos(eyes.x, eyes.y  + mc.player.getEyeHeight(), eyes.z, pos.x + 0.5 -  mc.getRenderManager().renderPosX, pos.y + 0.5 -  mc.getRenderManager().renderPosY, pos.z + 0.5 -  mc.getRenderManager().renderPosZ, red, green, blue, alpha);
+    }
+
+    private void drawLineFromPosToPos(double posx, double posy, double posz, double posx2, double posy2, double posz2, float red, float green, float blue, float opacity) {
```
Comment (dominikaaaa) [inline src/main/java/me/zeroeightsix/kami/module/modules/render/Search.java:294]: Could this go in render manager or KamiTesselator please? Also, the tracers module already has a from pos to player iirc

Comment (AFelidae) [reply src/main/java/me/zeroeightsix/kami/module/modules/render/Search.java:294]: Yeah should I update tracers too while im at it or just leave it be?

**Comment by dominikaaaa** [comment]
Go for it, the tracers code is a mess at this point (my bad) and lacking proper coloring

**Comment by dominikaaaa** [comment]
Sorry about the wait, there was a communication error and I was expecting someone else to merge it. Testing it right now, thanks for the pull!

**Comment by AFelidae** [comment]
All good, thanks :)

**Pull request merged**